### PR TITLE
show info modal on first visit

### PR DESCRIPTION
**Pull request by cwackerfuss**
Addresses https://github.com/cwackerfuss/react-wordle/issues/163

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,18 +17,23 @@ $> npm run start
...
 #### Production
+
 ```bash
 $> docker build --target=prod -t game:prod .
 $> docker run -d -p 80:80 game:prod
 ```
+
 Open [http://localhost](http://localhost) in browser.
 
 ## Projects built using this repo
```
Comment (cwackerfuss) [inline README.md:39]: Linter added these.

**Comment by TheKnowles** [comment]
LGTM!

File: README.md (hunk 1 above)
Comment (jtraverso) [reply README.md:39]: It would be good to add a time out on the alert so it takes somes ms in loading after page is loaded.

Comment (gheja) [reply README.md:39]: Is it possible to make a pull request to a pull request? I made the neccessary changes but don't want to mess up anything. I can also give suggestions here but only on files that are part of this PR but the `src/constants/settings.ts`  needs a change too.

**Pull request merged**